### PR TITLE
Adding Campaign Page Chips to Sections

### PR DESCRIPTION
**Pull request by dzhaniivanov**
#1430

### CODE REVIEW

File: src/components/client/campaigns/CampaignDetails.tsx [hunk 1]
```diff
@@ -121,6 +138,11 @@ export default function CampaignDetails({ campaign }: Props) {
         <Grid item xs={12}>
           <Divider />
         </Grid>
+        <Grid container item xs={12}>
+          {Object.entries(chipsLabel).map(([id, label]) => (
+            <CampaignDetailsChip key={id} chip={label} onClick={() => scrollToSection(id)} />
+          ))}
+        </Grid>
```
Comment (igoychev) [inline src/components/client/campaigns/CampaignDetails.tsx:145]: Looks good! Just let's move them directly below the title and also let's use color=primary ot make the main blue color of the site

Comment (dzhaniivanov) [reply src/components/client/campaigns/CampaignDetails.tsx:145]: I contact the ui/ux college to clarify translations and design(colors etc..)

Comment (dzhaniivanov) [reply src/components/client/campaigns/CampaignDetails.tsx]: Screenshot 2023-08-05 182515
Screenshot 2023-08-05 182453
i think is a good starting point(with some bugs for fixing)

**Review by igoychev — CHANGES_REQUESTED** [review]
the implementation is good, but the design needs to be updated, so let's wait for it

**Pull request closed without merging**